fieldservice_route_account/tests/test_dayroute_payment.py

```diff
@@ -0,0 +1,143 @@
+            'code': 'NC1110',
+            'name': 'Test Payable Account',
+            'user_type_id': self.user_type_payable.id,
+            'reconcile': True
+        })
+        self.user_type_receivable = self.env.ref(
+            'account.data_account_type_receivable')
+        self.account_receivable = self.env['account.account'].create({
+            'code': 'NC1111',
+            'name': 'Test Receivable Account',
+            'user_type_id': self.user_type_receivable.id,
+            'reconcile': True
+        })
+
+        # Create a customer
+        self.partner_customer_usd = self.Partner.create({
+            'name': 'Customer from the North',
+            'email': 'customer.usd@north.com',
+            'customer': True,
+            'property_account_payable_id': self.account_payable.id,
+            'property_account_receivable_id': self.account_receivable.id,
+        })
+
+        user_type_income = self.env.ref(
+            'account.data_account_type_direct_costs')
+        self.account_income_product = self.env['account.account'].create({
+            'code': 'INCOME_PROD111',
+            'name': 'Icome - Test Account',
+            'user_type_id': user_type_income.id,
+        })
+        # Create category
+        self.product_category = self.env['product.category'].create({
+            'name': 'Product Category with Income account',
+            'property_account_income_categ_id': self.account_income_product.id
+        })
+        uom_unit = self.env.ref('uom.product_uom_unit')
+        # Create a product
+        self.product_order = self.env['product.product'].create({
+            'name': "Zed+ Antivirus",
+            'standard_price': 235.0,
+            'list_price': 280.0,
+            'type': 'product',
+            'uom_id': uom_unit.id,
+            'uom_po_id': uom_unit.id,
+            'invoice_policy': 'order',
+            'expense_policy': 'no',
+            'default_code': 'PROD_ORDER',
+            'service_type': 'manual',
+            'taxes_id': False,
+            'categ_id': self.product_category.id,
+            'field_service_tracking': 'sale'
+        })
+        # Create a location
+        self.location = self.env.ref('fieldservice.location_1')
+        self.location.analytic_account_id = self.env.ref(
+            'analytic.analytic_administratif')
+        # Create a worker
+        self.fsm_worker = self.env['fsm.person'].create(
+            {'name': 'Test Worker'})
+        self.company = self.env.ref('base.main_company')
+        self.payment_method_manual_in = self.env.ref(
+            "account.account_payment_method_manual_in")
+        # Create a Journal
+        self.bank_journal = self.env['account.journal'].create(
+            {'name': 'Bank US', 'type': 'bank', 'code': 'BNK68',
+             'currency_id': self.company.currency_id.id})
+
+    def test_dayroute_account(self):
+        so = self.env['sale.order'].create({
```

Comment:
@murtuzasaleh This module does not depend on sale so you need to create invoices directly.

Reply:
Done